llvm/lib/Target/RISCV/RISCVInstrInfo.cpp

```diff
@@ -2463,10 +2462,6 @@ bool RISCVInstrInfo::verifyInstruction(const MachineInstr &MI,
         return false;
       }
     }
-    if (!RISCVII::hasSEWOp(TSFlags)) {
```

Comment:
Should we check `hasSEWOp && !hasVLOperand` that would throw an error SEW operand without VL operand.